lib/property_sets.rb

```diff
@@ -7,13 +7,17 @@
 rescue LoadError
 end
 
+if "#{ActiveRecord::VERSION::MAJOR}.#{ActiveRecord::VERSION::MINOR}" == "6.1"
+  ActiveRecord::Base.singleton_class.alias_method :connection_class_for_self, :connection_classes
```

Comment:
Where is `.connection_class_for_self` defined on other versions of Rails?

Reply:
It’s not defined, so we would fall back to `ActiveRecord::Base` in 6.0.

Reply:
But… what about Rails 7 then? You define the alias only for Rails 6.1, right?

Reply:
Ah, gotcha. Thanks.